distributedlog-benchmark/pom.xml

```diff
@@ -27,12 +27,12 @@
   <dependencies>
     <dependency>
       <groupId>org.apache.distributedlog</groupId>
-      <artifactId>distributedlog-client</artifactId>
+      <artifactId>distributedlog-proxy-client</artifactId>
```

Comment:
this change was based on #132. github doesn't allow pull request based on the other branch.